bitnami/elasticsearch/templates/configmap-initscripts.yaml

```diff
@@ -0,0 +1,15 @@
+{{- if and (or (.Files.Glob "files/docker-entrypoint-initdb.d/*.sh") .Values.initdbScripts) (not .Values.initdbScriptsConfigMap) }}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ template "elasticsearch.fullname" . }}-init-scripts
+  labels: {{- include "elasticsearch.labels" . | nindent 4 }}
+    component: master
+data:
+{{- if and (.Files.Glob "files/docker-entrypoint-initdb.d/*.sh") (not .Values.initdbScriptsConfigMap) }}
```

Comment:
After seeing several issues wit the use of the `files/` section, we preferred to not using that subfolder as an option.

Reply:
OK! Removed